test/modules/menue/components/toolBox/ToolBox.test.js

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable no-undef */
+
+import { ToolBox } from '../../../../../src/modules/menue/components/toolBox/ToolBox';
+import { toolBoxReducer } from '../../../../../src/modules/menue/store/toolBox.reducer';
+import { toggleToolBox } from '../../../../../src/modules/menue/store/toolBox.action';
+import { TestUtils } from '../../../../test-utils';
+import { $injector } from '../../../../../src/injection';
+
+window.customElements.define(ToolBox.tag, ToolBox);
+
+
+describe('ToolBoxElement', () => {
+
+	const setup = async (config = {}) => {
+
+		const { embed = false } = config;
+
+		const state = {
+			toolBox: {
+				open: true
+			}
+		};
+		TestUtils.setupStoreAndDi(state, { toolBox: toolBoxReducer });
+		$injector
+			.registerSingleton('EnvironmentService', {
+				isEmbedded: () => embed
+			})
+			.registerSingleton('SearchResultProviderService', { getGeoresourceSearchResultProvider: () => { } });
+		return TestUtils.render(ToolBox.tag);
+	};
+
+
+	describe('when initialized', () => {
+		it('adds a div which holds the toolbox', async () => {
+
+			const element = await setup();
+			expect(element.shadowRoot.querySelector('.tool-box.is-open')).toBeTruthy();
+			expect(element.shadowRoot.querySelector('.action-button')).toBeTruthy();
+
```

Comment:
We should also add tests for the content of the toolbox, 
e.g. is there a button for measurement